package.json

```diff
@@ -10,13 +10,12 @@
   "author": "Nordic Semiconductor ASA",
   "license": "SEE LICENSE IN LICENSE",
   "engines": {
-    "nrfconnect": "2.x"
+    "nrfconnect": "^2.4.0"
```

Comment:
OK, this specifies the required core version. Then discard the comment above. From this version, programmer will no longer supports previous versions of core.